src/main/java/org/opentripplanner/standalone/config/buildconfig/NetexConfig.java

```diff
@@ -164,6 +165,17 @@ private static NetexFeedParameters.Builder mapFilePatternParameters(
           .summary("Ignore contents of the FareFrame")
           .docDefaultValue(base.ignoreFareFrame())
           .asBoolean(base.ignoreFareFrame())
+      )
+      .withcodeTagKey(
+        config
+          .of("codeTagKey")
+          .since(V2_3)
+          .summary(
+            "Which NeTEx KeyValue should be looked on for the source of matching stops to platforms and stops, in addition to the ID."
+          )
+          .description("[Detailed documentation](BoardingLocations.md)")
```

Comment:
```suggestion
          .summary(
            "Which NeTEx `KeyValue` should be used for extracting a stop code."
          )
          .description("""
          The stop code is a GTFS concept but it can be used to cross-reference railway platforms and bus stops with data from OpenStreetMap to improve boarding locations without interfering with the stop ID which might be needed for other purposes.
          
          [Detailed documentation](BoardingLocations.md)""")
```